native/src/main/java/io/ballerina/stdlib/oauth2/OAuth2Client.java

```diff
@@ -104,20 +103,18 @@ public static Object doHttpRequest(BString url, BMap<BString, Object> clientConf
             } catch (Exception e) {
                 return createError("Failed to init SSL context. " + e.getMessage());
             }
-        }
+        }        
         HttpClient client = buildHttpClient(httpVersion);
         return callEndpoint(client, request);
     }
 
     private static URI buildUri(String url, BMap<BString, ?> secureSocket) throws IllegalArgumentException {
         String[] urlParts = url.split(OAuth2Constants.SCHEME_SEPARATOR, 2);
         if (urlParts.length == 1) {
-            urlParts = secureSocket != null ? new String[]{OAuth2Constants.HTTPS_SCHEME, urlParts[0]} :
+            urlParts = (secureSocket!=null) ? new String[]{OAuth2Constants.HTTPS_SCHEME, urlParts[0]} :
```

Comment:
Any specific reason to change this line?

Reply:
isPresent was a function of Optional container and was removed hence had to change this line too